javalib/src/main/scala/java/lang/System.scala

```diff
@@ -108,6 +107,71 @@ private object SystemProperties {
     systemProperties0
   }
 
+  private final val CurrentDirectoryKey = "user.dir"
+  private lazy val currentDirectory =
+    getCurrentDirectory().foreach(
+      systemProperties.setProperty(CurrentDirectoryKey, _)
+    )
+
+  private final val UserHomeDirectoryKey = "user.home"
+  private lazy val userHomeDirectory =
+    getUserHomeDirectory().foreach(
+      systemProperties.setProperty(UserHomeDirectoryKey, _)
+    )
+
+  private final val UserCountryKey = "user.country"
+  private lazy val userCountry =
+    getUserCountry().foreach(systemProperties.setProperty(UserCountryKey, _))
+
+  private final val UserLanguageKey = "user.language"
+  private lazy val userLanguage =
+    getUserLanguage().foreach(systemProperties.setProperty(UserLanguageKey, _))
```

Comment:
The result is always `Unit` so it might be confusing when reading source code. Maybe let's rename these variables so that they start with `init` prefix. 

I like the usage of lazy val - it would make sure that initialization is synchronized 


```suggestion
  private lazy val initUserLanguage =
    getUserLanguage().foreach(systemProperties.setProperty(UserLanguageKey, _))
```

Reply:
Went with the `initialize` prefix